[0m[[0m[31merror[0m] [0m[0mjava.lang.IndexOutOfBoundsException: -1 is out of bounds (min 0, max 9)[0m
[0m[[0m[31merror[0m] [0m[0m	at ... ()[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.RAT.$anonfun$new$50(rename.scala:247)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.Data.$anonfun$$colon$eq$1(Data.scala:718)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.experimental.prefix$.apply(prefix.scala:33)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.Data.$colon$eq(Data.scala:718)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.RAT.$anonfun$new$49(rename.scala:247)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.WhenContext.<init>(When.scala:131)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.WhenContext.elsewhen(When.scala:108)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.RAT.$anonfun$new$44(rename.scala:246)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.RAT.$anonfun$new$44$adapted(rename.scala:243)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.immutable.Range.foreach(Range.scala:190)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.RAT.$anonfun$new$43(rename.scala:243)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:190)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.RAT.<init>(rename.scala:242)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.rename.$anonfun$RAT$2(rename.scala:376)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.Module$.evaluate(Module.scala:88)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.Module$.do_apply(Module.scala:35)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.rename.$anonfun$RAT$1(rename.scala:376)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.internal.plugin.package$.autoNameRecursively(package.scala:33)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.rename.<init>(rename.scala:376)[0m
[0m[[0m[31merror[0m] [0m[0m	at ChaosCore.Main$.$anonfun$new$3(top.scala:104)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.Module$.evaluate(Module.scala:88)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.Module$.do_apply(Module.scala:35)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.stage.phases.Elaborate.$anonfun$transform$2(Elaborate.scala:48)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.internal.Builder$.$anonfun$buildImpl$1(Builder.scala:1023)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:59)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.internal.Builder$.buildImpl(Builder.scala:1017)[0m
[0m[[0m[31merror[0m] [0m[0m	at chisel3.internal.Builder$.$anonfun$build$1(Builder.scala:1008)[0m
[0m[[0m[31merror[0m] [0m[0m	at logger.Logger$.$anonfun$makeScope$4(Logger.scala:148)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:59)[0m
[0m[[0m[31merror[0m] [0m[0m	at logger.Logger$.makeScope(Logger.scala:146)[0m
[0m[[0m[31merror[0m] [0m[0m	at logger.Logger$.makeScope(Logger.scala:133)[0m
[0m[[0m[31merror[0m] [0m[0m	at ... ()[0m
[0m[[0m[31merror[0m] [0m[0m	at ... (Stack trace trimmed to user code only. Rerun with --full-stacktrace to see the full stack trace)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) java.lang.IndexOutOfBoundsException: -1 is out of bounds (min 0, max 9)[0m
